### PR TITLE
Add sortStart event

### PR DESCRIPTION
**Pull request by steel**
@thoughtless @adam-h @m1foley any objections?

### CODE REVIEW

File: dist/sordid-dragon.jquery.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Copyright Poll Everywhere
 // Paul Cortens, Mike Foley and Adam Heath
 // https://github.com/polleverywhere/sordid-dragon
-// Version 2.0.0
+// Version 2.0.1
```
Comment (thoughtless) [inline dist/sordid-dragon.jquery.js]: Something went wrong with the version number when this was generated. It should be `2.1.0`.

File: README.md [hunk 2]
```diff
@@ -51,6 +51,15 @@ To register an event handler that fires when a sort/drag is complete, include th
       }
     });
 
+To register an event handler that fires when a sort/drag starts, include the `sortStart` options:
+
+    $(".list").sordidDragon({
+      sortStart: function(event, child) {
+        // event is the raw JavaScript events
+        // child is a jQuery object referencing the child element that was just moved.
```
Comment (thoughtless) [inline README.md]: I suggest `child is a jQuery object referencing the child element that is being moved.`

**Review by thoughtless — COMMENTED** [review]
It would be great if you added this event to one of the examples. Since drag/drop code is rather hard to write automated tests for, we have done our testing with the files in the examples directory. Maybe do something that changes the background color of the page while something is being dragged, then changes it back in the `sortEnd` event.

**Comment by adam-h** [comment]
Added this to https://github.com/polleverywhere/sordid-dragon/pull/5

**Pull request merged**